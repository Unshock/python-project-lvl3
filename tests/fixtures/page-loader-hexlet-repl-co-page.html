<!DOCTYPE html>
<html lang="ru">
 <head>
  <meta charset="utf-8"/>
  <title>
   Курсы по программированию Хекслет
  </title>
  <link href="/page/assets/application.css" media="all" rel="stylesheet"/>
  <link href="/page/courses" rel="canonical"/>
 </head>
 <body>
  <img alt="Иконка профессии Node.js-программист" src="/page/assets/professions/nodejs.png"/>
  <h3>
   <a href="/page/professions/nodejs">
    Node.js-программист
   </a>
  </h3>
 </body>
 <script src="https://page-loader.hexlet.repl.co/page/packs/js/script1.js">
 </script>
 <script src="https://another.page-loader.hexlet.repl.co/page/packs/js/script1.js">
 </script>
 <script src="packs/js/script2.js">
 </script>
 <script src="../script3.js">
 </script>
 <script src="">
 </script>
</html>
