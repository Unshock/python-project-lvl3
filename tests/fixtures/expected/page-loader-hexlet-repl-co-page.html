<!DOCTYPE html>
<html lang="ru">
 <head>
  <meta charset="utf-8"/>
  <title>
   Курсы по программированию Хекслет
  </title>
  <link href="page-loader-hexlet-repl-co-page_files/page-loader-hexlet-repl-co-page-assets-application.css" media="all" rel="stylesheet"/>
  <link href="page-loader-hexlet-repl-co-page_files/page-loader-hexlet-repl-co-page-courses.html" rel="canonical"/>
 </head>
 <body>
  <img alt="Иконка профессии Node.js-программист" src="page-loader-hexlet-repl-co-page_files/page-loader-hexlet-repl-co-page-assets-professions-nodejs.png"/>
  <h3>
   <a href="/page/professions/nodejs">
    Node.js-программист
   </a>
  </h3>
 </body>
 <script src="page-loader-hexlet-repl-co-page_files/page-loader-hexlet-repl-co-page-packs-js-script1.js">
 </script>
 <script src="https://another.page-loader.hexlet.repl.co/page/packs/js/script1.js">
 </script>
 <script src="page-loader-hexlet-repl-co-page_files/page-loader-hexlet-repl-co-page-packs-js-script2.js">
 </script>
 <script src="page-loader-hexlet-repl-co-page_files/page-loader-hexlet-repl-co-script3.js">
 </script>
 <script src="">
 </script>
</html>
